cdap-api/src/main/java/co/cask/cdap/api/data/stream/StreamBatchReadable.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2014-2015 Cask Data, Inc.
+ * Copyright © 2014-2016 Cask Data, Inc.
```

Comment:
Opened a JIRA to track this change. It can happen separately from the deprecation. Else, this PR will get much larger.
https://issues.cask.co/browse/CDAP-5224